pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/ZkBasedTableRebalanceObserver.java

```diff
@@ -98,7 +98,8 @@ private void updateOnStart(Map<String, Map<String, String>> currentState,
 
   @Override
   public void onSuccess(String msg) {
-    Preconditions.checkState(_tableRebalanceProgressStats.getStatus() != RebalanceResult.Status.DONE.toString(),
+    Preconditions.checkState(
```

Comment:
intellij was complaining that this comparison was not correct. but i'm happy to undo this change if we don't want to touch this